media/stylus/wiki.styl

```diff
@@ -64,6 +64,7 @@ Styling for article content
 @require 'components/wiki/content/moreinfo';
 @require 'components/wiki/content/summary';
 @require 'components/wiki/customcss';
+@require 'components/wiki/samples';
```

Comment:
Yes, separate file++++ 

Can we name the file open-in-host instead of samples, please.